### ISSUE TITLE
/.kube/config: no such file or directory

### ISSUE THREAD

**Issue by Tomasz-Kluczkowski** [omrikiei/ktunnel#33]
Hi,

I have followed the example for Pycharm debugging.

I have deployed the example to docker-desktop kubernetes.

When I look into the logs on the `pyremotedebug-<hash>` pod I can see this error:

```
ktunnel time="2020-12-09T16:47:50Z" level=error msg="Failed getting kubernetes config: stat /.kube/config: no such file or directory"
ktunnel INFO[2020-12-09 16:47:50.669] Starting to listen on port 28688
```

I follow all the steps in the instructions , start the remote debugger, it is waiting for connection.
I do `ktunnel inject deployment pyremotedebug 4321`

and the output is:

```
INFO[0000] Injecting tunnel sidecar to default/pyremotedebug
WARN[0000] quay.io/omrikiei/ktunnel:latest already injected to the deplyoment
INFO[0000] Waiting for deployment to be ready
INFO[0000] Waiting for port forward to finish
INFO[0000] Forwarding from 127.0.0.1:28688 -> 28688
Forwarding from [::1]:28688 -> 28688
```

So I am missing the crucial bit shown in the instructions: `INFO[0011] starting tcp tunnel from source 4321 to target 4321`

looks like the tunnel is never started...

I assume it is because it cannot find this kube config file?

This happens also if I set KUBECONFIG in my terminal (but I guess this does not matter as it is the container log reporting missing kube config not the local ktunnel....)

If I do:
`curl localhost:8000/debug`

then in the pod's logs I can see:
```
pyremotedebug INFO:root:Connecting to remote debugger on 127.0.0.1:4321
pyremotedebug Could not connect to 127.0.0.1: 4321
```

Help please? Is there a step in the instructions missing?

**Comment by omrikiei** [comment]
Hi @Tomasz-Kluczkowski,
Actually the `Failed getting kubernetes config: stat /.kube/config: no such file or directory` error is legitimate on the server since it should not read kubernetes configurations, I will make sure to remove this error from the server to avoid confusion..

It seems that there is a bug with inject that was introduced following a refactor of the code, I'm fixing it.

**Comment by omrikiei** [comment]
Hi @Tomasz-Kluczkowski, a fix was released in version `v1.3.2`
Thank you for your feedback! enjoy ktunnel
